nupm/utils/dirs.nu

```diff
@@ -0,0 +1,39 @@
+# Directories and related utilities used in nupm
+
+# Decault installation path for nupm packages
+export const DEFAULT_NUPM_HOME = ($nu.default-config-dir | path join "nupm")
+
+# Prompt to create $env.NUPM_HOME if it does not exist and some sanity checks.
+export def nupm-home-prompt [] {
+    if 'NUPM_HOME' not-in $env {
+        error make {
+            msg: "Internal error: NUPM_HOME environment variable is not set"
+        }
+    }
+
+    if ($env.NUPM_HOME | path exists) {
+        if ($env.NUPM_HOME | path type) != 'dir' {
+            error make {
+                msg: ($"Root directory ($env.NUPM_HOME) exists, but is not a"
```

Comment:
as you can see in the thread above, the words are missing some spaces with this string concatenation :wink: